coursebuilder/views/base.html

```diff
@@ -217,70 +217,61 @@
     <!-- footer section -->
 
     <!--  Footer  -->
-    <footer>
-      <div class="card footer-top">
-        <div class="row">
+<footer>
```

Comment:
@Sharonsyra Should the Indentation be aligned with <!-- Footer -->?

Reply:
Fixed.